### DISCUSSION TITLE
Disconnected Agents

### DISCUSSION THREAD

**Discussion by tfcmansoor** [wazuh/wazuh#19611] [Q&A]
One Of My Endpoints Is Alive, However In WAZUH - It is Showing Disconnected Agent.

How To Solve This Problem. Any Help ?

**Comment by pereyra-m** [comment]
Hello @tfcmansoor !

To properly assist you, please provide more details:

- What is your Wazuh version for both manager and agent?
- Can you upload `ossec.conf` and `ossec.log` files from both?
- How did you register this agent?

Regards.

**Comment by pereyra-m** [comment]
Hi again @tfcmansoor !

Please, provide more details so we can help you.

Regards.

**Comment by pereyra-m** [comment]
@tfcmansoor , I'll close this discussion due to the lack of activity, but feel free to open it again if you need to.
